src/OpenLoco/Window.cpp

```diff
@@ -764,6 +764,39 @@ namespace OpenLoco::Ui
         }
     }
 
+    void window::viewportFromSavedView(const SavedViewSimple& savedView)
+    {
+        auto viewport = viewports[0];
+        if (viewport != nullptr)
+        {
+            auto& config = viewport_configurations[0];
+            config.viewport_target_sprite = EntityId::null;
+            config.saved_view_x = savedView.mapX;
+            config.saved_view_y = savedView.mapY;
+
+            auto zoom = static_cast<int32_t>(savedView.zoomLevel) - viewport->zoom;
+            if (zoom != 0)
+            {
+                if (viewport->zoom < 0)
```

Comment:
```
../src/OpenLoco/Window.cpp: In member function 'void OpenLoco::Ui::window::viewportFromSavedView(const OpenLoco::Ui::SavedViewSimple&)':
../src/OpenLoco/Window.cpp:780:36: error: comparison is always false due to limited range of data type [-Werror=type-limits]
                 if (viewport->zoom < 0)
                     ~~~~~~~~~~~~~~~^~~
```

Reply:
Should probably be zoom not viewport zoom